EarthCubeInGeo/resen
# [BUG]: Fix bug with update command

This PR fixes #73 and adds documentation about the `update` command. I've also bumped the release version numbers in preparation of a quick minor release.

TESTING
----------

To test this, in the same resen session:
1) update
2) create

Without this PR, you'll see that you can't access the latest 2020.2.0 resen-core without restarting resen in between steps 1 and 2.

DOCS
------
You can view the update to the docs here: https://resen.readthedocs.io/en/bugfix_update/usage.html#update-list-of-available-cores
